include/reflection-cpp/reflection.hpp

```diff
@@ -145,7 +145,6 @@ namespace detail
 } // namespace detail
 
 template <class T, class... Args>
-    requires(std::is_aggregate_v<std::remove_cvref_t<T>>)
```

Comment:
True. this was just a test. Because I was lost. Truth is, it compiles all very well locally on my Fedora, including GCC 14. Github actions is also GCC 14. This is confusing me.